spec/takeway_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'takeaway'
+
+describe Takeaway do
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+  it 'should show list of dishes with prices' do
```

Comment:
you could assign the dishes hash to a variable and call it where you need it

Reply:
dishes = { "rice" => 1.25, "banana" => 1.00, "fries" => 2.10 }

subject(:takeaway) { Takeaway.new(dishes) }

then just do your tests without having to create a new takeaway object each time